contracts/votingMachines/GenesisProtocolLogic.sol

```diff
@@ -112,12 +109,6 @@ contract GenesisProtocolLogic is IntVoteInterfaceEvents {
         uint256 _amount
     );
 
-    event RedeemReputation(bytes32 indexed _proposalId,
```

Comment:
Subgraph is indexing this. Better to either leave it (or if you don't want older daos to show these rewards anymore we can drop it entirely from the subgraph)